app/blog/index.html

```diff
@@ -0,0 +1,34 @@
+---
+layout: default
+title: MVRT Build Blog
+---
+<!-- This loops through the paginated posts -->
+{% for post in site.posts limit: 5 %}
+	<div class="post-preview">
+		<h2><a class="post-title" href="{{ post.url }}">{{ post.title }}</a></h2>
+		<span class="post-date">{{ post.date | date: "%B %d, %Y" }}</span>
+		<div class="post-content">
+			{{ post.content | split:'<!--break-->' | first }}
+		</div>
+ 
+		{% if post.content contains '<!--break-->' %}
+			<a class="read-more" href="{{ post.url }}">read more</a>
```

Comment:
Jekyll [provides a `post.excerpt` variable][0] that splits the post based on an `excerpt_separator` site variable. Or an `excerpt` can be set in the YAML front matter. So no need to split by `<!--break-->`, I guess.

[0]: http://jekyllrb.com/docs/posts/#post-excerpts